lib/engine/game/g_1812/step/loan_operations.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+require_relative '../../g_1867/step/loan_operations'
+
+module Engine
+  module Game
+    module G1812
+      module Step
+        class LoanOperations < G1867::Step::LoanOperations
+          def skip!
+            pass!
+            entity = current_entity
+
+            owed = @game.pay_interest!(entity)
+            if owed
+              @game.unpaid_loan(entity, owed)
```

Comment:
Refactoring `G1867::Step::LoanOperations` to extract a (private) `unpaid_interest` method called at this point would mean that only that method would need to be redone in this class, instead of copying all of the code in this method.

Reply:
@ollybh Thanks I must have missed/forgotten about this one. 

I'm pretty sure I get what you're saying. I submitted a new commit, have a look.